app/models/donation.rb

```diff
@@ -0,0 +1,7 @@
+class Donation < ActiveRecord::Base
+  belongs_to :donor
+  has_one :cause
+
+  validates :amount, presence: true
```

Comment:
Let's also validate the presence of a `Donor`, using `validates :donor, presence: true`. In addition, we should validate that `amount` is a number greater than zero.

On a general note, let's group our validations by attribute, rather than by validation type, so in this case:

```
validates :donor, presence: true
validates :amount, presence: true, numericality: { greater_than: 0 }
```

rather than:

```
validates :donor, :amount, presence: true
validates :amount, numericality: { greater_than: 0 }
```